docs/03_register.md

```diff
@@ -0,0 +1,32 @@
+# Different gating strategy
```

Comment:
let's add a link to https://maci.pse.dev/docs/developers-references/smart-contracts/Gatekeepers